hadoop-ozone/dev-support/intellij/ozone-site.xml

```diff
@@ -59,12 +59,12 @@
     <name>hdds.prometheus.endpoint.enabled</name>
     <value>true</value>
   </property>
-  <property>
-    <name>ozone.recon.address</name>
-    <value>localhost:9891</value>
-  </property>
-  <property>
-    <name>ozone.recon.db.dir</name>
-    <value>/tmp/recon</value>
-  </property>
+<!--  <property>-->
+<!--    <name>ozone.recon.address</name>-->
+<!--    <value>localhost:9891</value>-->
+<!--  </property>-->
+<!--  <property>-->
+<!--    <name>ozone.recon.db.dir</name>-->
+<!--    <value>/tmp/recon</value>-->
+<!--  </property>-->
```

Comment:
I think this is unintended change in this PR.

Reply:
Sorry for that